mysql-test/main/mysql_json_mysql_upgrade_system_tables.result

```diff
@@ -0,0 +1,92 @@
+#
```

Comment:
Not a big fan of the test name. Can we have two commits please of the form:

1. Rename this test as mysql_upgrade_mysql_json_upgrade_system_tables (in this commit)
2. Rename all mysql_json_mysql_upgrade_xxx tests to mysql_upgrade_mysql_json_xxx (in a separate commit)

This should make things consistent with the other test names in main suite related to mysql_upgrade (or rather mariadb-upgrade)